editor/filesystem_dock.cpp

```diff
@@ -2209,15 +2209,15 @@ void FileSystemDock::_file_option(int p_option, const Vector<String> &p_selected
 		case FILE_DEPENDENCIES: {
 			// Checkout the file dependencies.
 			if (!p_selected.is_empty()) {
-				String fpath = p_selected[0];
+				const String &fpath = p_selected[0];
 				deps_editor->edit(fpath);
 			}
 		} break;
 
 		case FILE_OWNERS: {
 			// Checkout the file owners.
 			if (!p_selected.is_empty()) {
-				String fpath = p_selected[0];
+				const String &fpath = p_selected[0];
 				owners_editor->show(fpath);
```

Comment:
```suggestion
				owners_editor->show(p_selected[0]);
```

Reply:
I'm biased against this change -- `fpath = <...>` is a pattern used across `filesystem_dock.cpp` -- I think it adds code patterning and readability to the maintainer.